package.json

```
@@ -43,7 +42,8 @@
     "coverage": "codecov",
     "docs:dev": "docz dev",
     "docs:build": "docz build",
-    "contributors:add": "all-contributors add"
+    "contributors:add": "all-contributors add",
+    "install-16": "npm install --save-dev react@\"^16.9.0\" react-dom@\"^16.9.0\" react-test-renderer@\"^16.9.0\""
```

a convenience script to install React 16.9, can be modelled though to install any version of React

---

Using the `^` in the install command will allow npm to install the later minor version that matches that range, so I think you'll likely end up with the latest version of 16 if you run that.

I'd be tempted to use `--no-save` instead of `--save-dev` so that switch the version for testing doesn't modify the `package.json` and then needing to remembering to revert it before committing.

I'd also include one for the other supported versions (`@^16`, `@^17`).  Something like:

```suggestion
    "install:react-16-9": "npm install --no-save react@16.9.0 react-dom@16.9.0 react-test-renderer@16.9.0",
    "install:react-16": "npm install --no-save react@^16 react-dom@^16 react-test-renderer@^16",
    "install:react-17": "npm install --no-save react@^17 react-dom@^17 react-test-renderer@^17"
```

---

Great idea! Thanks!